### PR TITLE
Feature/1138 articleupload

### PR DESCRIPTION
**Pull request by richard-jones**
Review version of the article ingest code.  This moves most of the article ingest functionality into the task and handles the possible asynchronous file system.

There is a set of tests now for all this functionality, but it will be good to test this on the dev server like a real user would use it.

### CODE REVIEW

File: portality/view/publisher.py [hunk 1]
```diff
@@ -92,16 +93,26 @@ def upload_file():
...
     
     if url is not None and url != "":
-        return _url_upload(url, schema, previous)
+        flash("File reference successfully received - it will be processed shortly", "success")
+        return render_template('publisher/uploadmetadata.html', previous=previous)
     
     flash("No file or URL provided", "error")
     return render_template('publisher/uploadmetadata.html', previous=previous)
 
+"""
```
Comment (emanuil-tolev) [inline portality/view/publisher.py]: Is this code supposed to be commented out? It is actually modified below in a couple of places but the whole function is inactive.

Comment (richard-jones) [inline portality/view/publisher.py]: Yes, it's not needed, just left it in for reference at the time, it can now be removed.

**Comment by emanuil-tolev** [comment]
Noted that we need the s3fs set up on test ASAP to finish this one now.

**Comment by emanuil-tolev** [comment]
Also I need the huey always eager code for my tests - I have actually copied it over into my branch, but can now do a merge from develop, so thanks for that.

**Pull request merged**